api/tests/unit-tests/schemas/test_evaluation.py

```diff
@@ -102,14 +105,16 @@ def test_EvaluationRequest():
     # test missing args
     with pytest.raises(ValidationError):
         schemas.EvaluationRequest(
-            model_filter=None,  # type: ignore - purposefully throwing error
-            datum_filter=schemas.Filter(),
+            dataset_names=["ds"],
+            model_names=None,  # type: ignore - purposefully throwing error
+            filter=schemas.Filter(),
             parameters=schemas.EvaluationParameters(
                 task_type=enums.TaskType.CLASSIFICATION
             ),
         )
     with pytest.raises(ValidationError):
         schemas.EvaluationRequest(
+            dataset_names=["ds"],
```

Comment:
this test should probably check if dataset_names is `None` since `datum_filter` is no longer an attribute  of `EvaluationRequest`. can you ctrl + F for "datum_filter" in this file and make sure we aren't using it inappropriately on other tests?